controllers/account.js

```diff
@@ -1,12 +1,18 @@
+const { cloudinary } = require("../cloudinary");
+
 const { accountStyle } = require("../public/javascripts/extraStyles.js");
 const { fetchProfileData } = require("../middleware/helpers.js");
 const User = require("../models/user");
 
+const defaultProfileImg =
+  "https://res.cloudinary.com/dyguovdbc/image/upload/w_400/v1680099865/pixelify/PixelatedVergil_rl9cpv.jpg";
```

Comment:
This same string is declared in /middleware/helpers.js as well. Let's declare once and reuse it.